src/app/(components)/Navbar.tsx

```diff
@@ -0,0 +1,16 @@
+import Menu from "./Menu"
+import "./Navbar.css" 
+import Cart from "./Cart"
+import React from "react"
+
+const Navbar = ()=>{
+    return(
+        <div className="navbarContainer">
+            <Menu/> 
+            <img  src="/Screenshot 2024-02-06 143929.png" alt="Wituka Image" className="logoImage"/>
```

Comment:
You did well here, Creating the Cart component and using an image. Next,js comes with an Image component that optimizes the images for you. Let's use it

```suggestion
//Top of the file
import Image from "next/image";  

// Instead of the img tag
            <Image  src="/Screenshot 2024-02-06 143929.png" alt="Wituka Image" className="logoImage"/>
```